packages/relay/tests/lib/eth/filter.spec.ts

```diff
@@ -0,0 +1,98 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import path from 'path';
+import dotenv from 'dotenv';
+import MockAdapter from 'axios-mock-adapter';
+import { expect } from 'chai';
+import { Registry } from 'prom-client';
+import { MirrorNodeClient } from '../../../src/lib/clients/mirrorNodeClient';
+import pino from 'pino';
+import constants from '../../../src/lib/constants';
+import { ClientCache } from '../../../src/lib/clients';
+import { FilterService } from '../../../src/lib/services/ethService';
+
+dotenv.config({ path: path.resolve(__dirname, '../test.env') });
+
+const logger = pino();
+const registry = new Registry();
+
+let restMock: MockAdapter, web3Mock: MockAdapter;
+let mirrorNodeInstance: MirrorNodeClient;
+let filterService: FilterService;
+let clientCache: ClientCache;
+let mirrorNodeCache: ClientCache;
+
+
+describe('Filter API Test Suite', async function () {
```

Comment:
nit: something to think on and doesn't have to be fixed now but we should consider if we want the test structure to follow the src structure